hl_extractor/hl_calc.py

```diff
@@ -62,21 +64,27 @@ def _calculate(self):
             subprocess.check_call([os.path.join(".", HIGH_LEVEL_EXTRACTOR_BINARY),
                                    name, out_file, PROFILE_CONF],
                                   stdout=fnull, stderr=fnull)
-        except subprocess.CalledProcessError:
+        except (subprocess.CalledProcessError, OSError):
             print("Cannot call high-level extractor")
+            # If we return early, make sure we remove the temp
+            # output file that we created
+            os.unlink(out_file)
             return "{}"
-
-        fnull.close()
-        os.unlink(name)
+        finally:
+            # At this point we can remove the source file,
+            # regardless of if we failed or if we succeeded
+            fnull.close()
```

Comment:
hmm, so we do 
```python
with open(devnull) as fnull:
    try:
        subprocess.check_call(, , fnull)
    except:
        blah
```
?
It seems a little odd to use it just for this file handle. Although, I guess all a file context manager does is close the file handle in a finally block (https://docs.python.org/2/library/contextlib.html#contextlib.closing). By doing it ourselves we save a level of indentation.